thriftbp/prometheus.go

```diff
@@ -255,3 +255,12 @@ func stringifyErrorType(err error) string {
 	}
 	return strings.TrimPrefix(fmt.Sprintf("%T", err), "*")
 }
+
+// We create this interface instead of using errors.go/Error because we want it
```

Comment:
I think you meant to say "baseplateError" instead of "errors.go/Error"? I'm not sure what "errors.go/Error" is.

Reply:
updated